utility/SPIDEV/gpio.h

```diff
@@ -41,19 +41,17 @@ class GPIO
 
     GPIO();
 
-    static void open(int port, int DDR);
+    static void Gopen(int port, int DDR);
 
-    static void close(int port);
+    static void Gclose(int port);
 
-    static int read(int port);
+    static int Gread(int port);
 
-    static void write(int port, int value);
+    static void Gwrite(int port, int value);
```

Comment:
Why did these get renamed?

Reply:
There was some sort of conflict with the names, I was getting errors if I left the names as-is:
```
/home/pi/rf24libs/RF24/utility/SPIDEV/gpio.cpp: In static member function 'static void GPIO::open(int, int)':
/home/pi/rf24libs/RF24/utility/SPIDEV/gpio.cpp:32:15: error: invalid conversion from 'const char*' to 'int' [-fpermissive]
   32 |     fd = open(dev_name, O_RDONLY);
      |               ^~~~~~~~
      |               |
      |               const char*
/home/pi/rf24libs/RF24/utility/SPIDEV/gpio.cpp:30:21: note:   initializing argument 1 of 'static void GPIO::open(int, int)'
   30 | void GPIO::open(int port, int DDR)
      |                 ~~~~^~~~
/home/pi/rf24libs/RF24/utility/SPIDEV/gpio.cpp:32:14: error: void value not ignored as it ought to be
   32 |     fd = open(dev_name, O_RDONLY);
      |          ~~~~^~~~~~~~~~~~~~~~~~~~
/home/pi/rf24libs/RF24/utility/SPIDEV/gpio.cpp:38:19: error: invalid conversion from 'const char*' to 'int' [-fpermissive]
   38 |         fd = open(dev_name, O_RDONLY);
```

Reply:
That should be solved be explicitly specifying the global namespace:
```c++
fd = ::open(dev_name, O_RDONLY);
```
Otherwise calling `open()` within the context of `GPIO::open()` makes the compiler think it is a recursive call to `GPIO::open()`. Using `::open()` avoids that.

Reply:
Never heard of that before, but it works! Done.

Reply:
I also pushed a commit to change `close()` to `::close()` just to be safe.